src/components/irregularity/admin/grid/IrregularityFile.tsx

```diff
@@ -52,15 +60,9 @@ export default function IrregularityFile({ file, irregularityId }: Props) {
       </ListItemAvatar>
       <ListItemText primary={file.filename} />
       <></>
-      <Tooltip
-        title={
-          'Note: This link is public on the API side! Need to correct before official release!'
-        }>
-        <Button>
-          {/* TODO: to be discussed. Tracked in issue: https://github.com/podkrepi-bg/frontend/issues/811 */}
-          <a style={{ color: 'red' }} href={API_URL + `/irregularity-file/${file.id}`}>
-            {t('admin.cta.download') + '*'}
-          </a>
+      <Tooltip title={'download'}>
+        <Button style={{ color: 'red' }} onClick={downloadFileHandler}>
+          {t('admin.cta.download') + '*'}
```

Comment:
I guess we don't need the asterisk anymore as there's no comment

```suggestion
          {t('admin.cta.download')}
```